### ISSUE TITLE
Interactive training - formatting console output for action names and slots

### ISSUE THREAD

**Issue by rojobuffalo** [RasaHQ/rasa_core#419]
**Rasa Core version**: 0.8.5

**Python version**: 3.6.5

**Operating system** (windows, osx, ...): OSX

**Issue**: The interface for interactive training is incredibly good. Being able to write lots of good training data for a bot is obviously very important for improving a bot's intelligence. I'm getting bottlenecked in the same 2 places in the training process: 1) finding action numbers in the list of actions which gets presented when you are correcting the action decision; 2) scanning slots in the part that says `we currently have slots:`. 

https://imgur.com/a/Cz0Rk40
https://imgur.com/a/5Rg5N5N

These are just segments of the larger lists, so this information is not easy to direct the eye to. I'm wondering if we can either change the text layout or have it be configurable. I would want to left-align the names of actions because I think that would be a lot easier to read and scan to the corresponding number id. Then I would want to print slots on their own line so they could also be easier read and scanned.

Is this a change worth considering?

**Comment by rojobuffalo** [comment]
Tagging the file which would be changed:
https://github.com/RasaHQ/rasa_core/blob/20889d10d0596a579a8b2b22e86ca29e91e8f91a/rasa_core/policies/online_policy_trainer.py

**Comment by akelad** [comment]
hmm i'm not sure. I mean, if we printed the slots each on a seperate line and there's a lot of slots, this would mean lots of scrolling if you want to look at the intent recognised and stuff. Also I'm not sure left aligning the actions would help much, because then i guess you'd have the same problem with searching for the accuracy.

**Comment by rojobuffalo** [comment]
I didn't initially consider that for different projects or training sessions you might end up needing to focus on different areas. For the one I'm working on currently, reading the names of actions and checking which slots are filled is something I'm doing often; and I would benefit from having that information be less compact in the log. But now that I'm imagining what you bring up, I get it that the current configuration works well in some cases. Like once the bot is working pretty well and you're pretty familiar with the numbers for next actions, you would spend more time looking at the accuracy numbers.

Maybe instead of a hard-coded change, could we consider a configurable option? My first thoughts for potential way of doing that:
- `--history-format-slots-delimiter ["\n", ", "]`
- `--history-format-slots ["expanded", "compact"]`
- `--log-format-slots-delimiter ["\n", ", "]`
- `--log-format-next-action-align ["left", "right"]`
- `--next-action-list-align ["left", "right"]`

Idk if this good naming or even a good idea, just throwing it out to see what you think. I would be happy to write the change and PR if ya'll were warm to the idea.

**Comment by tmbo** [comment]
@rojobuffalo I don't think we should make this configurable and agree that the information can be displayed better. How about you change it in a way that works best for you and we work from there? e.g. we could also write two slots per line (capping the outputed value at n chars) or something like that.

**Comment by rojobuffalo** [comment]
@tmbo ok, I'll play around with it.
